server/queries/users.sql

```diff
@@ -0,0 +1,29 @@
+-- PUT/:uid - Updates an existing user as approved
+UPDATE
+    users
+SET
+    approved = TRUE
+WHERE
+    id = ?;
+
+-- DELETE/:uid - Deletes an existing user given their id
+DELETE FROM
+    users
+WHERE
+    id = ?;
+
+-- GET - Returns all data from the Users table
+SELECT * FROM
+    Users;
```

Comment:
good work! just a minor fix: the `users` table is lower case (sorry for the inconsistency in the directions). but other than that, the code looks good!

Reply:
oh okay got it I was confused about that, thanks for clarifying.